azkaban-common/src/main/java/azkaban/executor/ExecutionFlowDao.java

```diff
@@ -277,6 +293,19 @@ private void updateExecutableFlow(final ExecutableFlow flow, final EncodingType
     }
   }
 
+  private void updateExecutableFlowStatus(final ExecutableFlow flow)
```

Comment:
Nit: Consider renaming this function to "updateExecutableFlowStatusInDB". Looking at the current name I thought the function will update the status in flow object.

Reply:
Will do